### PR TITLE
Amir/static pages solution

### PR DESCRIPTION
**Pull request by amir-deriv**
# Description

- Static pages working solution with Automated translation workflow using Crowdin

### CODE REVIEW

File: crowdin.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+project_id_env: "CROWDIN_PROJECT_ID"
+api_token_env: "CROWDIN_PERSONAL_TOKEN"
+base_path_env: "CROWDIN_BASE_PATH"
+base_url_env: "CROWDIN_BASE_URL"
```
Comment (ali-hosseini-deriv) [inline crowdin.yml:4]: these have to be static? is it right?

File: example.env [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+CROWDIN_PROJECT_ID=
+CROWDIN_PERSONAL_TOKEN=
+CROWDIN_BASE_PATH=
+CROWDIN_BASE_URL=
```
Comment (ali-hosseini-deriv) [inline example.env:5]: EOF

File: prettier.config.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "htmlWhitespaceSensitivity": "css",
+    "vueIndentScriptAndStyle": false,
+    "proseWrap": "preserve",
+    "insertPragma": false,
+    "requirePragma": false,
+    "tabWidth": 2,
+    "useTabs": false,
+    "embeddedLanguageFormatting": "auto",
+    "printWidth": 120
+}
```
Comment (ali-hosseini-deriv) [inline prettier.config.json:20]: EOF

File: templates/signup/index.html [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+            </button>
+        </div>
+    </div>
+    <footer class="mt-12">
+        <h1 id="ls-test-feature-flag" class="text-2xl text-center text-[#414652]">This is Default</h1>
+    </footer>
+
+</body>
+
+</html>
```
Comment (ali-hosseini-deriv) [inline templates/signup/index.html:112]: EOF

**Review by ali-hosseini-deriv — COMMENTED** [review]
some nitpicks.

File: crowdin.yml (hunk 1 above)
Comment (amir-deriv) [reply crowdin.yml:4]: Yes, it is. it's just mapping the environment variables in the action

File: .github/workflows/generate_and_deploy_pages.yml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
+    steps:
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
```
[the remark this answers is not in the document]
Comment (amir-deriv) [reply .github/workflows/generate_and_deploy_pages.yml:26]: not sure, I copied from deriv-app action. removed it now

**Review by ali-hosseini-deriv — APPROVED** [review]

**Pull request merged**